salt/metalk8s/addons/prometheus-operator/deployed/service-configuration.sls

```diff
@@ -0,0 +1,41 @@
+#! metalk8s_kubernetes
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metalk8s-monitoring
+  name: metalk8s-grafana-config
+data:
+  config.yaml: |-
+    apiVersion: addons.metalk8s.scality.com
+    kind: GrafanaConfig
+    spec:
+      deployment:
+        replicas: 1
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metalk8s-monitoring
+  name: metalk8s-prometheus-config
+data:
+  config.yaml: |-
+    apiVersion: addons.metalk8s.scality.com
+    kind: PrometheusConfig
+    spec:
+      deployment:
+        replicas: 1
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metalk8s-monitoring
+  name: metalk8s-alertmanager-config
+data:
+  config.yaml: |-
+    apiVersion: addons.metalk8s.scality.com
+    kind: AlertmanagerConfig
+    spec:
+      deployment:
+        replicas: 1
```

Comment:
IMO we should do some jinja here to check that these ConfigMap not already exists to not override customers changes if this state run (and then when we have this, this state could be included in the `init.sls` of each addons)

Reply:
Done!